editor/src/core/vscode/vscode-bridge.ts

```diff
@@ -146,10 +155,17 @@ export async function initVSCodeBridge(
       await clearBothMailboxes()
       await writeProjectContents(projectID, projectContents)
       await initMailbox(UtopiaInbox, parseFromVSCodeMessage, (message: FromVSCodeMessage) => {
-        dispatch(
-          [selectFromFileAndPosition(message.filePath, message.line, message.column)],
-          'everyone',
-        )
+        switch (message.type) {
+          case 'SEND_INITIAL_DATA':
+            dispatch([sendCodeEditorInitialisation()], 'everyone')
+            break
+          case 'EDITOR_CURSOR_POSITION_CHANGED':
+            dispatch(
+              [selectFromFileAndPosition(message.filePath, message.line, message.column)],
+              'everyone',
+            )
+            break
```

Comment:
could you add our usual catchall `_exhaustiveCheck` here?

Reply:
Fixed in 65d34bc6.